src/routers/city.js

```diff
@@ -0,0 +1,11 @@
+const {Router} = require('express');
+const router = Router();
+const {getCities, postCity} = require('../controllers/city-controller');
+const {validateExistState} = require('../controllers/state-controller');
+const {validationStateId, validationCity}= require('../validators/city-validator');
+
+router.get('/cities/:stateID', validationStateId, getCities);
+
+router.post('/cities', validationCity, validateExistState, postCity);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
router.post("/cities", validationCity, validateExistState, postCity);
```